### ISSUE TITLE
chown: invalid group: 'myuser:myuser'

### ISSUE THREAD

**Issue by Kube-ASY** [kubermatic/kubeone#444]
**What happened**:
I tried to install a new KubeOne cluster with the following config:
```
apiVersion: kubeone.io/v1alpha1
kind: KubeOneCluster
name: demo
versions:
  kubernetes: "1.13.2"
cloudProvider:
  name: "none"
hosts:
- publicAddress: '1.2.3.4'
  privateAddress: '172.18.0.1'
  sshPort: 22 # can be left out if using the default (22)
  sshUsername: ubuntu
machineController:
  deploy: false
  provider: "none"
```
This failed with the following issue:
```
~/go/bin/kubeone install config.yaml -v
INFO[15:47:26 CEST] Installing prerequisites…
INFO[15:47:26 CEST] Determine operating system…                   node=172.18.0.1
[172.18.0.1] + command . /etc/os-release
[172.18.0.1] + NAME='CentOS Linux'
[172.18.0.1] + VERSION='7 (Core)'
[172.18.0.1] + ID=centos
[172.18.0.1] + ID_LIKE='rhel fedora'
[172.18.0.1] + VERSION_ID=7
[172.18.0.1] + PRETTY_NAME='CentOS Linux 7 (Core)'
[172.18.0.1] + ANSI_COLOR='0;31'
[172.18.0.1] + CPE_NAME=cpe:/o:centos:centos:7
[172.18.0.1] + HOME_URL=https://www.centos.org/
[172.18.0.1] + BUG_REPORT_URL=https://bugs.centos.org/
[172.18.0.1] + CENTOS_MANTISBT_PROJECT=CentOS-7
[172.18.0.1] + CENTOS_MANTISBT_PROJECT_VERSION=7
[172.18.0.1] + REDHAT_SUPPORT_PRODUCT=centos
[172.18.0.1] + REDHAT_SUPPORT_PRODUCT_VERSION=7
[172.18.0.1] + echo -n centos
[172.18.0.1] centos
INFO[15:47:26 CEST] Determine hostname…                           node=172.18.0.1
[172.18.0.1] + hostname -f
[172.18.0.1] exunxlvh06404
[172.18.0.1] + fqdn=exunxlvh06404
[172.18.0.1] + [ exunxlvh06404 '=' localhost ]
[172.18.0.1] + echo exunxlvh06404
INFO[15:47:26 CEST] Creating environment file…                    node=172.18.0.1
[172.18.0.1] + sudo mkdir -p /etc/kubeone
[172.18.0.1] + cat
[172.18.0.1] + sudo tee /etc/kubeone/proxy-env
[172.18.0.1] + 0<< \EOF
[172.18.0.1]
[172.18.0.1] EOF
[172.18.0.1]
INFO[15:47:26 CEST] Installing kubeadm…                           node=172.18.0.1 os=centos
[172.18.0.1] + sudo swapoff -a
[172.18.0.1] + sudo sed -i '/.*swap.*/d' /etc/fstab
[172.18.0.1] + sudo sed -i 's/SELINUX=enforcing/SELINUX=permissive/g' /etc/sysconfig/selinux
[172.18.0.1] + command . /etc/kubeone/proxy-env
[172.18.0.1] + 1> /dev/null
[172.18.0.1] + exit 0
[172.18.0.1] + whence -v docker
[172.18.0.1] docker is a tracked alias for /usr/bin/docker
INFO[15:47:26 CEST] Deploying configuration files…                node=172.18.0.1 os=centos
[172.18.0.1] + sudo mkdir -p /etc/systemd/system/kubelet.service.d/ /etc/kubernetes
[172.18.0.1] + sudo mv ./kubeone/cfg/cloud-config /etc/kubernetes/cloud-config
[172.18.0.1] + sudo chown root:root /etc/kubernetes/cloud-config
[172.18.0.1] + sudo chmod 600 /etc/kubernetes/cloud-config
INFO[15:47:26 CEST] Generating kubeadm config file…
INFO[15:47:26 CEST] Configuring certs and etcd on first controller…
INFO[15:47:26 CEST] Ensuring Certificates…                        node=172.18.0.1
+ [[ -d ./kubeone/pki ]]
+ sudo kubeadm init phase certs all '--config=./kubeone/cfg/master_0.yaml'
W0513 15:47:26.844573   11322 common.go:86] WARNING: Detected resource kinds that may not apply: [InitConfiguration JoinConfiguration]
[config] WARNING: Ignored YAML document with GroupVersionKind kubeadm.k8s.io/v1beta1, Kind=JoinConfiguration
[certs] Using certificateDir folder "/etc/kubernetes/pki"
[certs] Using existing ca certificate authority
[certs] Using existing apiserver-kubelet-client certificate and key on disk
[certs] Using existing apiserver certificate and key on disk
[certs] Using existing front-proxy-ca certificate authority
[certs] Using existing front-proxy-client certificate and key on disk
[certs] Using existing etcd/ca certificate authority
[certs] Using existing etcd/healthcheck-client certificate and key on disk
[certs] Using existing apiserver-etcd-client certificate and key on disk
[certs] Using existing etcd/server certificate and key on disk
[certs] Using existing etcd/peer certificate and key on disk
[certs] Using the existing "sa" key
+ mkdir -p ./kubeone/pki/etcd
+ sudo cp /etc/kubernetes/pki/ca.crt ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/ca.key ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/sa.key ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/sa.pub ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/front-proxy-ca.crt ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/front-proxy-ca.key ./kubeone/pki/
+ sudo cp /etc/kubernetes/pki/etcd/ca.crt /etc/kubernetes/pki/etcd/ca.key ./kubeone/pki/etcd/
+ sudo chown -R myuser:myuser ./kubeone
chown: invalid group: 'myuser:myuser'
WARN[15:47:27 CEST] Task failed, retrying…
```

**What is the expected behavior**:
Installation proceeds.

**How to reproduce the issue**:
Try to install.

**Anything else we need to know?**

**Information about the environment**:
KubeOne version (kubeone version): v0.6.0-4-g532c39b
Operating system: CentOS Linux release 7.6.1810 (Core)
Provider you're deploying cluster on: Bare-metal
Operating system you're deploying on: CentOS Linux release 7.6.1810 (Core)

**Comment by kron4eg** [comment]
@Kube-ASY thanks for report!

I don't see it, from where `myuser` comes from. Your sshUsername is `ubuntu` (on a CentOS machine??) but somehow myuser is in the error message. Do you have any ideas how's that possible?

**Comment by xmudrii** [comment]
@Kube-ASY Thank you for reporting the issues!

We've merged two PRs, #439 and #443, which should fix issues you reported. We also just released KubeOne v0.6.2 which includes those fixes. Please let us know if there are any other issues. Thanks!

**Comment by Kube-ASY** [comment]
Hi, I am sorry about that. I don't use ubuntu or myuser as the user but my real username. I manually replaced both the user in the config and in the output.
